src/OmniSharp.Abstractions/OmniSharp.Abstractions.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="2.8.2" />
```

Comment:
I think this breaks layering. IIRC, `OmniSharp.Abstractions` was not supposed to have a dependency on Roslyn. @david-driscoll, is that right?

Reply:
that was definitely the idea, that's also why `OmniSharp.Roslyn` is there, as an extra layer over `OmniSharp.Roslyn.CSharp`